src/utils.ts

```diff
@@ -4,7 +4,28 @@ import type {
   CommonBlockProps,
   FileBlockProps,
   FolderBlockProps,
-} from "@utils";
+} from "../utils/types";
```

Comment:
For some reason, the build kept failing when the `@utils` alias was present. The only thing I can think of is that the compiler got confused by `utils.ts` requiring a directory called `utils` 🤷 

As an aside, maybe it's weird to have both a `utils.ts` file that lives outside a completely separate `utils` directory with totally different stuff inside? I'm sure if I git blame this, I was probably the one who did this 😛